client/src/views/Home.js

```diff
@@ -29,7 +30,9 @@ export default function Home() {
 
           <div id="Inscrieri" className=" pt-0">
           <ButoanePrincipale/>
-          {/* Act. secundare */}
+
+          <ActivitatiSecundare/>
+          
```

Comment:
you still have some newlines that don't make sense, you should delete them manually if your priettier doesn't delete them